content/blog/2016-10-29-unix-tools-for-windows/index.md

```diff
@@ -9,7 +9,9 @@ categories:
     - GMT工具
 tags:
     - Linux
-    - Windows
+    - Windows    
+redirect: https://docs.gmt-china.org/latest/install/windows/#id1
```

Comment:
This URL with the anchor "#id1" is fragile and it can link to other places if we change the Windows installation section. We should add a new label in https://github.com/gmt-china/GMT_docs/blob/master/source/install/windows.rst and use the new link here.

Reply:
Updated in 404af71.